bootstrap-app/src/cmr/bootstrap/migrations/003_drop_quartz_tables.clj

```diff
@@ -1,7 +1,5 @@
 (ns cmr.bootstrap.migrations.003-drop-quartz-tables
-  (:require [clojure.java.jdbc :as j]
-            [config.bootstrap-migrate-config :as config]
-            [config.bootstrap-migrate-helper :as h]))
+  (:require [config.bootstrap-migrate-helper :as h]))
```

Comment:
I think our style guide tells us to put the files on the next line using 1 space such as
```
(:require
 [clojure.string :as string]
```